src/fides/api/util/consent_util.py

```diff
@@ -350,6 +360,11 @@ def create_privacy_notices_util(
                 model=privacy_notice,
                 fields=PRIVACY_NOTICE_ESCAPE_FIELDS,
             )
+        privacy_notice = transform_fields(
```

Comment:
this feels a little clunky - but i wasn't finding an obvious way to do this "natively" in the pydantic model. this "manual" transformation doesn't deviate too much from the existing pattern we have for escaping.

Reply:
(see comment below - different transform, not escaping!)

Reply:
thanks for clarifying I missed this! 🔍   do any of the gpp field mappings sub fields need to be added to PRIVACY_NOTICE_ESCAPE_FIELDS. For example, gpp_field_mapping > notice.  This would complicate things though not saying we need it.

Reply:
ah, i don't think so, i think we're good! the strings there should just be basic uppercase/lowercase alphabet characters. obviously wouldn't hurt but i don't think we need to build in that complexity for now 👍